src/Microsoft.AspNetCore.Sockets/ConnectionManager.cs

```diff
@@ -56,22 +56,26 @@ public bool TryGetConnection(string id, out DefaultConnectionContext connection)
             return _connections.TryGetValue(id, out connection);
         }
 
-        public DefaultConnectionContext CreateConnection()
+        public DefaultConnectionContext CreateConnection(PipeOptions pipeOptions)
         {
             var id = MakeNewConnectionId();
 
             _logger.CreatedNewConnection(id);
             var connectionTimer = SocketEventSource.Log.ConnectionStart(id);
+            var pair = DuplexPipe.CreateConnectionPair(pipeOptions, pipeOptions);
 
-            var pair = DuplexPipe.CreateConnectionPair(PipeOptions.Default, PipeOptions.Default);
-            
             var connection = new DefaultConnectionContext(id, pair.Application, pair.Transport);
             connection.ConnectionTimer = connectionTimer;
 
             _connections.TryAdd(id, connection);
             return connection;
         }
 
+        public DefaultConnectionContext CreateConnection()
```

Comment:
Do we need this overload? It's an internal type, so I think we don't need it.

Reply:
I believe the tests use it.